graph-builder/src/main.rs

```diff
@@ -43,17 +43,20 @@ fn main() -> Result<(), Error> {
 
     let state = graph::State::new();
     let addr = (opts.address, opts.port);
-    let graph_path = format!("/{}/v1/graph", opts.path_namespace);
+    let app_prefix = opts.path_prefix.clone();
 
     {
         let state = state.clone();
         thread::spawn(move || graph::run(&opts, &state));
     }
 
     server::new(move || {
-        App::with_state(state.clone())
+        let app_prefix = app_prefix.clone();
```

Comment:
Can this (and the `state` below too) be just moved in? it looks like there is nothing else holding references to those.